pkg/webhook/ingress/validating.go

```diff
@@ -56,74 +59,80 @@ func (w *webhook) GetPath() string {
 	return "/validating-ingress"
 }
 
-type handler struct{}
+type handler struct {
+	Log logr.Logger
+}
 
 func Handler() capsulewebhook.Handler {
-	return &handler{}
+	return &handler{Log: ctrl.Log.WithName("controllers").WithName("Tenant")}
```

Comment:
Adding a logger for logging purposes during debugging.

Reply:
No problem, let's remove it once ready to get it merged: there's a better way to debug webhooks, although pretty difficult to setup.

https://github.com/clastix/capsule/blob/master/contributing.md#run-capsule-locally